backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -448,6 +456,12 @@ public void deleteOrganization(Organization org) {
 
   // returns the external ID of the organization the specified user belongs to
   public Optional<OrganizationRoleClaims> getOrganizationRoleClaimsForUser(String username) {
+    // when accessing tenant data, bypass okta and get org from the altered authorities
+    if (_tenantDataContextHolder.hasBeenPopulated()
+        && username.equals(_tenantDataContextHolder.getUsername())) {
```

Comment:
**thought:** Given that the TenantDataContextHolder is request-scoped, it seems like `getOrganizationRoleClaimsForUser`'s expected contract won't accurately apply Tenant Data Access to users _other than_ the logged-in user making the request.

**question:** Is this a problem? Should the LiveOktaRepository depend on the TenantDataAccessRepository for this specific logic instead?

Reply:
It may be, as we discussed, but the use case for now makes it unlikely that a site admin would be requesting the claims for another site admin.  If we need to, we can revisit this later and return the tenant data access claims for other site admins.